deploifai/project/create.py

```diff
@@ -68,14 +69,19 @@ def create(context: DeploifaiContextObj, name: str, workspace):
         click.echo("An error occurred when fetching projects. Please try again.")
         return
 
-    err_msg = "Project name taken. Choose a unique project name:"
-
     project_names = [project["name"] for project in projects]
+    user_pwd_dir_names = os.listdir()
+
+    dup_local_err_msg = f"There are existing files/directories in your computer also named {name}"
+    dup_backend_err_msg = f"Project name taken. Existing names in chosen workspace: {' '.join(project_names)}\nChoose a unique project name:"
 
-    name_taken_err_msg = f"Project name taken. Existing names in chosen workspace: {' '.join(project_names)}\nChoose a unique project name:"
-    err_msg = name_taken_err_msg
+    err_msg = ""
+    if name in user_pwd_dir_names:
+        err_msg = dup_local_err_msg
+    elif name in project_names:
+        err_msg = dup_backend_err_msg
 
-    is_valid_name = not (name in project_names)
+    is_valid_name = not (name in project_names or name in user_pwd_dir_names)
```

Comment:
Just set `is_valid_name` to `true` under each conditional when setting the error message. Code becomes less buggy that way.